gradle/findbugs-exclusions.xml

```diff
@@ -10,4 +10,13 @@
   <Match>
     <Class name="~org\.eclipse\.golo\.compiler\.parser.*" />
   </Match>
+  <Match>
+    <!-- Ignore naming conventions for methods and classes used in golo code -->
+    <Class name="~gololang.*" />
+    <Bug code="Nm" />
+  </Match>
+  <Match>
+    <Class name="gololang.Unknown" />
```

Comment:
It's a special singleton added for special matching of union types, to represent values you don't care about (like _ in Caml and other languages). e.g.
```golo
let _ = Unknown: get()
union Tree = {
  Node = {left, right}
  Leaf = {value}
  Empty
}

match {
  when tree: isEmpty() then "empty tree"
  when tree: isLeaf(0) then "a leaf with 0"
  when tree: isLeaf() then "a leaf"
  when tree: isNode(Tree.Empty(), _) or tree: isNode(_, Tree.Empty()) then "node with 1 child"
  when tree: isNode() then "a node"
  otherwise "wtf"
}
```

To work, the instance of `Unknown` is equals to anything but null. But findbugs does not like that, so I mask it out.